### PR TITLE
Add example with Dask Bag

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: bag.ipynb [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import dask\n",
+    "import json\n",
+    "import os\n",
+    "\n",
+    "os.makedirs('data', exist_ok=True)              # Create data/ directory\n",
+    "\n",
+    "b = dask.datasets.make_people()                 # Make records of people\n",
```
[the remark this answers is not in the document]
Comment (mrocklin) [reply bag.ipynb]: Resolved

File: bag.ipynb [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df[df.age > 30].occupation.value_counts().nlargest(10).compute()"
```
[the remark this answers is not in the document]
Comment (mrocklin) [reply bag.ipynb]: Resolved

File: bag.ipynb [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Start Dask Client for Dashboard\n",
+    "\n",
+    "Starting the Dask Client is optional.  It will provide a dashboard which \n",
+    "is useful to gain insight on the computation.  \n",
+    "\n",
+    "The link to the dashboard will become visible when you create the client below.  We recommend having it open on one side of your screen while using your notebook on the other side.  This can take some effort to arrange your windows, but seeing them both at the same is very useful when learning."
```
[the remark this answers is not in the document]
Comment (mrocklin) [reply bag.ipynb]: I did not address this comment.  My apologies.  This comment is in most of the notebooks.  You're right that we should probably change it a bit now that we have the JLab extension.  I'd like to do this across all of the notebooks consistently though.

[the remark this answers is not in the document]
Comment (guillaumeeb) [reply bag.ipynb]: That surely makes sense!

**Pull request merged**